utils/search.go

```diff
@@ -24,3 +24,20 @@ func ContainsString(val string, vals []string) bool {
 	}
 	return false
 }
+
+func TextToKeywords(in string) string {
+	s := []byte(in)
+	j := 0
+	for _, b := range s {
+		if ('a' <= b && b <= 'z') ||
+			('A' <= b && b <= 'Z') ||
+			('0' <= b && b <= '9') {
```

Comment:
Should we apply more general unicode rules? Or are we restricting values to ascii?
```suggestion
	for _, b := range in {
		if unicode.IsLetter(b) || unicode.IsDigit(b) {
```

Reply:
unicode is fine, thanks